info-articles/introduction-to-decentralized-storage/Article.md

```diff
@@ -0,0 +1,142 @@
+
+- **Encryption:** Storj encrypts files using the AES-256-GCM symmetric encryption. This is routine procedure for every file before it is uploaded to the network, which ensures that no unauthorized person has access to your data. Only those who have been allowed access can view your data.
+
+- **Splitting:** Each file is divided into 80 pieces, and accessing a file requires only 29 of those components. Each piece is stored on a separate Node, each having its own set of operators, power sources, networks, and geographies.
+
+- **Distribution:** Each component of an object is dispersed across a vast global network of nodes, ensuring that data is never concentrated in a single location.
+
+- **Retrieveing:** Storj only need 29 encrypted fragments to get a file. The network can automatically recreate the data contained in the remaining fragments using only those 29 components.
+
+With these principles, Storj boasts of being immune to censorship, data loss, malicious attacks, and service failures
+encryption:
+
+**Cost of storage:**
+0.004$/GB/Month
+
+### Arweave
+
+Arweave is another decentralized storage network that aspires to dethrone major cloud storage providers such as Google. It was formerly named Archain. With more than 500 nodes in operation, the goal of Arweave is to store files over a distributed network of computers permanently.
+
+Storage space sellers on the Arweave Network get paid in the network's native $AR coin. This gives them an incentive to keep such files available for clients. It works on two layers which are:
+
+**Blockweave:** Arweave's data is stored on a block graph. In Arweave, each block is linked to two previous blocks, generating a structure known as a "blockweave."
+
+**Permaweb:** Everything published on the permaweb is permanently accessible. The permaweb provides low-cost, zero-maintenance hosting for their web apps and online pages.
+
+**Cost of storage:**
+Approx $8 one-time lifetime storage cost/GB. No monthly subscription
+
+### Bluzelle
+
+Bluzelle is a decentralized database storage network driven by Cosmos and its Byzantine fault-tolerant  (BFT) technology Tendermint, which allows it to be compatible with several blockchains.
+
+Bluzelle provides a decentralized framework for financial data by merging decentralized technologies with edge computing, ensuring that organizations never experience data breaches, network outages, or poor performance, all at a price of a few pennies. Bluzzelle boasts of a lot of features like data immutability, speed, lower cost, high privacy e.t.c
+
+Bluzelle users can make money by renting out their computer storage space. Developers, academics, content providers, and others can rent this space for significantly less than the cost of using centralized cloud services like Amazon's AWS. The cost savings can be used to improve the titles and development processes. In the end, everyone benefits from a more efficient and profitable process.
+
+## Conclusion
```

Comment:
Hey @oyedeletemitope,
I went through your article, and I like that you covered all of the major decentralized storage systems.
I think the list of systems covered is quite exhaustive.

One thing I feel could be added is that decentralized storage systems are not brand new. Think of UTorrent and BitTorrent. Could you add a couple of lines on these?

I will forward your article internally in the meantime, and at the same time, I will be looking forward to your edit.

Reply:
Hello @Genesis3800 . I’ll work on that straight away and get back to you asap.